Library/Formula/flex.rb

```diff
@@ -1,13 +1,13 @@
 class Flex < Formula
   homepage "http://flex.sourceforge.net"
-  url "https://downloads.sourceforge.net/flex/flex-2.5.37.tar.bz2"
-  sha1 "db4b140f2aff34c6197cab919828cc4146aae218"
+  url "https://downloads.sourceforge.net/flex/flex-2.5.39.tar.bz2"
+  sha1 "267794c709d5c50f2dcb48ff5d8dcbbfe40f953e"
 
   bottle do
     revision 1
-    sha1 "0a2bb0ce9a49330e5fd40b6e409a353972cf8840" => :yosemite
-    sha1 "0dd7fc9c36a6258b2e456e7dd0c5818d07e2a2ea" => :mavericks
-    sha1 "69b5f449a9c0bf5fd37f999dca5ccfd120a6f389" => :mountain_lion
+    sha1 "a0595f788aa425fec3f225e200d029e33785a98e" => :yosemite
+    sha1 "1c8d3b0d0e0ffcc5ccfa27508c01acb2f385b38b" => :mavericks
+    sha1 "f21aeb00534e0ac4fad94a8635b4be5f0e7a31d2" => :mountain_lion
   end
```

Comment:
You can leave the bottle stanza alone; we have automation for updating it when we pull.